client/app/components/packages/rwcds-form/proposed-project-actions-editor.hbs

```diff
@@ -1,39 +1,62 @@
 {{#let @form as |Form|}}
```

Comment:
I know this isn't part of the diff, but can we change this to lowercase `|form|` here, too?

Reply:
for sure!